src/data/equiv/basic.lean

```diff
@@ -758,6 +758,93 @@ def sigma_equiv_prod_of_equiv {α β} {β₁ : α → Sort*} (F : Π a, β₁ a
 
 end
 
+section prod_congr
+
+variables {α₁ β₁ β₂ : Type*} (e : α₁ → β₁ ≃ β₂)
+
+/-- A family of equivalences `Π (a : α₁), β₁ ≃ β₂` generates an equivalence
+between `β₁ × α₁` and `β₂ × α₁`. -/
+def prod_congr_left : β₁ × α₁ ≃ β₂ × α₁ :=
```

Comment:
Should this be 
```suggestion
@[congr] def prod_congr_left : β₁ × α₁ ≃ β₂ × α₁ :=
```
to match `prod_congr`?

Reply:
Hmm, I can't figure out how to test that change. I thought `@[congr]` is used by the `congr` and `simp` tactics, but neither of these two work:

```lean
example : ℕ × ℤ ≃ ℤ × ℤ := by { congr, show ℕ ≃ ℤ, sorry } -- congr tactic failed
example : ℕ × ℤ ≃ ℤ × ℤ := by { have : ℕ ≃ ℤ := sorry, simp } -- simplify tactic failed to simplify
```

Reply:
I've no idea either, sorry

Reply:
[I posted the question to Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/.60.40.5Bcongr.5D.20def.20prod_congr.60)

Reply:
Gabriel suggests not marking it as `congr`, so I'll follow that advice.